src/components/shared/Story/index.tsx

```diff
@@ -265,8 +328,13 @@ class ActivityContent extends React.Component<{
         </View>
         {/* TODO(tina): Render call to action for tipping
         {actorCallToAction &&
+<<<<<<< HEAD
         actorsData !== RAHA_BASIC_INCOME_MEMBER && ( 
+=======
+        actorsData !== RAHA_BASIC_INCOME_MEMBER && ( // TODOt
+>>>>>>> 58687245... Add tip UI
```

Comment:
merge conflict?

Reply:
ah oops i wasn't supposed to merge the UI changes into this PR

Reply:
just fixed the bad merge, it happened because I was switching back and forth between my UI branch and this one